teams.md/docs/main/teams/configuration/_category_.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I moved our configuration docs to a Configuration section.

Reply:
why not under "deployment" ?

Reply:
Because you actually need it not just for deployment, but even for use in Teams. Thoughts?

Reply:
got it